### PR TITLE
[3/N] Enable clang-tidy on torch/csrc/inductor

### PR DESCRIPTION
**Pull request by cyyever**
Follows #132040

### CODE REVIEW

**Comment by cyyever** [comment]
@Skylion007 Docker builds were trigged, interesting.

File: torch/csrc/inductor/aoti_runtime/model_container.h [hunk 1]
```diff
@@ -129,31 +128,31 @@ class AOTInductorModelContainer {
     if (this->num_models() == 0) {
       throw std::runtime_error("No available models in container!");
     }
-    return models_[0]->constant_name(idx);
+    return models_[0]->constant_name(static_cast<int64_t>(idx));
```
Comment (Skylion007) [inline torch/csrc/inductor/aoti_runtime/model_container.h:131]: This is a lot of casting, should constant_name just actually have a size_t type? Or atleast an overload for it?

**Review by Skylion007 — APPROVED** [review]

File: torch/csrc/inductor/aoti_runtime/model_container.h (hunk 1 above)
Comment (cyyever) [reply torch/csrc/inductor/aoti_runtime/model_container.h:131]: It can be, I will check that later

**Pull request closed without merging**